autopilot/contractor/contractor.go

```diff
@@ -355,10 +352,20 @@ func (c *Contractor) renewContract(ctx *mCtx, contract contract, host api.Host,
 	}
 
 	// calculate the expected new storage
-	expectedNewStorage := renterFundsToExpectedStorage(renterFunds, endHeight-cs.BlockHeight, pt)
+	var expectedNewStorage uint64
+	if host.IsV2() {
+		expectedNewStorage = renterFundsToExpectedStorageV2(renterFunds, contract.EndHeight()-cs.BlockHeight, host.V2Settings.Prices)
```

Comment:
`nit` Instead of having a `V2` counterpart for everything wouldn't it make more sense to pass in the host? Especially in this case because the parameters are virtually identical if you pass in the `host`.